tools/githooks/pre-push

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env vpython3
```

Comment:
Avoids issue with Windows setting up alias for python3 to the Microsoft Store by default. This will also make sure that every install uses the depot_tools python version.